torchvision/ops/ciou_loss.py

```diff
@@ -0,0 +1,94 @@
+import math
```

Comment:
safely drop this import after you remove math.pi

```suggestion

```

Reply:
There exists functions which depend on `math` module. 

https://github.com/pytorch/vision/blob/467b841ac2650c962952bbee5a542edb835c98bc/test/test_ops.py#L277-L278

But i see no problem in changing these  from `math.floor()` to `torch.floor()`. 
@oke-aditya

Reply:
I'm pointing to different file I guess. 
But good idea to replace there as well.